auth/auth_test.go

```diff
@@ -37,6 +37,7 @@ import (
 )
 
 var client *Client
+var ctx context.Context
```

Comment:
Is this ever assigned to? Personally, I think we are better off calling `context.Background()` everywhere this is needed. Otherwise it is somewhat confusing since this is referenced from multiple test files.

Reply:
This gets set in test main for the app engine in line 60, else 68. 
are we dropping the AE support?

Reply:
Ah I see. Then lets leave it as is for now. Going forward though I think we should remove the dev app server test mode. It's complicating the code here.